tests/unit/s2n_asn1_time_test.c

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 201 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include <stdlib.h>
+#include "s2n_test.h"
+#include "utils/s2n_asn1_time.h"
+
+int main(int argc, char **argv)
+{
+    BEGIN_TEST();
+
+    //October 20, 2017 3:09:11 PM GMT-07:00
```

Comment:
It's not your fault - we've become a bit sloppy about mixing C++ and C-style comments, I think I need to write a linter! But it's probably worth using just one comment style to avoid making it worse. 

PS: Am appropriately filled with self-loathing for giving comment style nit feedback.

Reply:
doh! Will fix.

Reply:
also I highly recommend clang tidy and clang rename. It will automatically do this for you.